hack/common.sh

```diff
@@ -124,6 +129,26 @@ os::build::host_platform() {
   echo "$(go env GOHOSTOS)/$(go env GOHOSTARCH)"
 }
 
+os::build::build_e2e_binaries() {
+    version_ldflags=$1
+    
+    # _test.go files cannot be used as dependencies, see https://golang.org/pkg/testing/ for reason on why.
+    # Then, our e2e_test.go  will use the packages created here.
+    E2E_TEST="Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/"
+    E2E_TEST_FILE="$E2E_TEST/e2e_test.go"
+    TMP_E2E_TEST_RENAMED="$E2E_TEST/e2e_test_copied.go"
+
+    # Copy the renamed file.
+    cp $E2E_TEST_FILE $TMP_E2E_TEST_RENAMED
```

Comment:
Yes this a little kludgey.  I think debating the single binary is better done in the issue, rather than here... 

One alternative impl is to copy the source code as a file we curate in openshift.